src/components/MarkdownProvider/components/CodeExample/utils/retrieveCodesandboxParameters.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { getParameters } from 'codesandbox/lib/api/define';
+
+const html = `
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="theme-color" content="#000000">
+	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+	<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+	<title>React App</title>
+</head>
+<body>
+	<noscript>
+		You need to enable JavaScript to run this app.
+	</noscript>
+	<div id="root"></div>
+</body>
+</html>
+`;
+
+const index = `
+import * as React from "react";
+import { render } from "react-dom";
+import { ThemeProvider } from "@zendeskgarden/react-theming";
+
+/** Some opinionated style resets for Zendesk products. Optional. */
+import '@zendeskgarden/css-bedrock';
+
+import Example from "./Example";
+
+const rootElement = document.getElementById("root");
+render(
+  <div style={{ padding: 16 }}>
+    <ThemeProvider>
+      <Example />
+    </ThemeProvider>
+  </div>,
+  rootElement
+);
+`;
+
+const packageJson = `
+{
+  "dependencies": {
+    "react-scripts": "latest",
+    "react": "latest",
+    "react-dom": "latest",
+    "styled-components": "latest",
+    "@zendeskgarden/css-bedrock": "^7.0.33",
+    "@zendeskgarden/css-variables": "^6.4.3",
```

Comment:
I don't think we want to include `css-variables`.

Reply:
Removed